<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LTWindow</class>
 <widget class="QMainWindow" name="LTWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>828</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>F1LT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/icons.qrc">
    <normaloff>:/ui_icons/icon.png</normaloff>:/ui_icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="NoSessionBoardWidget" name="messageBoardWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="EventStatusWidget" name="eventStatusWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="LTWidget" name="ltWidget" native="true"/>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Driver data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="DriverDataWidget" name="driverDataWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Session data</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="SessionDataWidget" name="sessionDataWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Weather charts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="WeatherChartsWidget" name="weatherChartsWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>201</red>
              <green>201</green>
              <blue>201</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>201</red>
              <green>201</green>
              <blue>201</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>201</red>
              <green>201</green>
              <blue>201</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>201</red>
              <green>201</green>
              <blue>201</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <attribute name="title">
         <string>Commentary</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>220</red>
                 <green>220</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>20</red>
                 <green>20</green>
                 <blue>20</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Highlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>159</red>
                 <green>159</green>
                 <blue>159</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>220</red>
                 <green>220</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>20</red>
                 <green>20</green>
                 <blue>20</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Highlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>159</red>
                 <green>159</green>
                 <blue>159</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>130</red>
                 <green>130</green>
                 <blue>130</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>240</red>
                 <green>240</green>
                 <blue>240</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Highlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>175</red>
                 <green>175</green>
                 <blue>175</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="lineWrapMode">
            <enum>QTextEdit::WidgetWidth</enum>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1249</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionOpen"/>
    <addaction name="actionLT_files_data_base"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSession_analysis"/>
    <addaction name="actionSession_times"/>
    <addaction name="actionDriver_tracker"/>
    <addaction name="actionFollow_a_driver"/>
    <addaction name="actionHead_to_head"/>
    <addaction name="actionLap_time_comparison"/>
    <addaction name="actionTrack_records"/>
    <addaction name="separator"/>
    <addaction name="actionRecord"/>
    <addaction name="actionStop_recording"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLT_files_data_base"/>
   <addaction name="actionOpen"/>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionFollow_a_driver"/>
   <addaction name="actionHead_to_head"/>
   <addaction name="actionLap_time_comparison"/>
   <addaction name="actionSession_analysis"/>
   <addaction name="actionSession_times"/>
   <addaction name="actionDriver_tracker"/>
   <addaction name="actionTrack_records"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="actionStop_recording"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>5</height>
    </size>
   </property>
  </widget>
  <action name="actionHead_to_head">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/h2h.png</normaloff>:/ui_icons/h2h.png</iconset>
   </property>
   <property name="text">
    <string>Head to head</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/login.png</normaloff>:/ui_icons/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Login</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/record.png</normaloff>:/ui_icons/record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/preferences.png</normaloff>
     <normalon>:/ui_icons/preferences.png</normalon>:/ui_icons/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/exit.png</normaloff>:/ui_icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/ui_icons/icon3.png</normaloff>:/ui_icons/icon3.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/document-open.png</normaloff>:/ui_icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLap_time_comparison">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/ltc.png</normaloff>:/ui_icons/ltc.png</iconset>
   </property>
   <property name="text">
    <string>Lap time comparison</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStop_recording">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/stop.png</normaloff>:/ui_icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop recording</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSession_analysis">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/sa.png</normaloff>:/ui_icons/sa.png</iconset>
   </property>
   <property name="text">
    <string>Session analysis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLT_files_data_base">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/ltdb.png</normaloff>:/ui_icons/ltdb.png</iconset>
   </property>
   <property name="text">
    <string>LT files database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFollow_a_driver">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/fad.png</normaloff>:/ui_icons/fad.png</iconset>
   </property>
   <property name="text">
    <string>Follow a driver</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSession_times">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/st.png</normaloff>:/ui_icons/st.png</iconset>
   </property>
   <property name="text">
    <string>Session times</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDriver_tracker">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/dt.png</normaloff>:/ui_icons/dt.png</iconset>
   </property>
   <property name="text">
    <string>Driver tracker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTrack_records">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/ui_icons/tr.png</normaloff>:/ui_icons/tr.png</iconset>
   </property>
   <property name="text">
    <string>Track records</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DriverDataWidget</class>
   <extends>QWidget</extends>
   <header>main_gui/driverdatawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SessionDataWidget</class>
   <extends>QWidget</extends>
   <header>main_gui/sessiondatawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EventStatusWidget</class>
   <extends>QWidget</extends>
   <header>main_gui/eventstatuswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoSessionBoardWidget</class>
   <extends>QWidget</extends>
   <header>main_gui/nosessionboardwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeatherChartsWidget</class>
   <extends>QWidget</extends>
   <header>main_gui/weatherchartswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LTWidget</class>
   <extends>QWidget</extends>
   <header>main_gui/ltwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered(bool)</signal>
   <receiver>LTWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
